scripts/split_repo_migration/delete-everything-split-repo.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script was written as part of a project to migrate
+# split repositories to a mono repository google-cloud-python.
+# This script will delete all files in a split repository and
+# update the README to indicate that the repository is archived.
+
+# `-e` enables the script to automatically fail when a command fails
+set -e
+
+# Ensure that both arguments are provided
+# Args:
```

Comment:
Consider moving lines 26 and 27 into the echo message, that way the user gets the info at run time and we document in a single spot.

Reply:
Fixed in f1c5fef32